app/src/ui/branches/branch-list-item.tsx

```diff
@@ -27,10 +29,24 @@ interface IBranchListItemProps {
   readonly onRenameBranch?: (branchName: string) => void
 
   readonly onDeleteBranch?: (branchName: string) => void
+
+  readonly onDropOntoBranch?: (branchName: String) => void
+}
+interface IBranchListItemState {
```

Comment:
Complete nit, sorry, but I'm missing here a new line above the `interface` 😳 
```suggestion

interface IBranchListItemState {
```